BackEndApp/BackEndApp/settings.py

```diff
@@ -124,4 +115,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
-STATIC_URL = '/static/'
+STATIC_URL = "/static/"
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+# activate less secure apps of Gmail Account
+EMAIL_HOST_USER = "you gmail id"
+EMAIL_HOST_PASSWORD = "your gmail password"
```

Comment:
Use `os.getenv("EMAIL")` and `os.getenv("EMAIL_PASSWORD")` or something on these lines :)